Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -24,6 +24,7 @@ static const u64 TEXHASH_INVALID = 0;
 static const int TEXTURE_KILL_THRESHOLD = 200;
```

Comment:
You likely also want to reduce this one. I guess 200 was only used at the time when we also remove EFB copys. I guess so ~10 frames should be fine.